src/Marten.Testing/Events/ScenarioAggregateAndRepository.cs

```diff
@@ -70,6 +73,23 @@ public void CanGuardVersion()
             // ENDSAMPLE
         }
 
+        [Fact]
+        public void CanRetrieveVersion()
+        {
+            var repository = new AggregateRepository(theStore);
+
+            var invoice = CreateInvoice();
+            invoice.Version.ShouldBe(3);
+            repository.Store(invoice);
+
+            // assert version was incremented properly
+            var invoiceFromRepository = repository.Load<Invoice>(invoice.Id);
+            invoiceFromRepository.Version.ShouldBe(3);
+
+            // update aggregate
+            invoiceFromRepository.AddLine(100, 23, "Some nice product with 23% VAT");
```

Comment:
Those scenarios are missing:
```
repository.Store(invoiceFromRepository);

            // assert version was incremented properly
            invoiceFromRepository = repository.Load<Invoice>(invoice.Id);
            invoiceFromRepository.Version.ShouldBe(4);
```
from my PR https://github.com/JasperFx/marten/pull/1299/files#diff-22165075f6da4b37fe6d31e83b98d376R91

Reply:
done, added it